src/main/python/pybuilder/plugins/python/core_plugin.py

```diff
@@ -63,11 +63,15 @@ def list_modules():
     project.list_modules = list_modules
 
     def list_scripts():
+        def is_hidden_file(file_path):
+            return file_path.startswith('.')
+
         scripts_dir = project.expand_path("$dir_source_main_scripts")
         if not os.path.exists(scripts_dir):
             return
         for script in os.listdir(scripts_dir):
-            if os.path.isfile(os.path.join(scripts_dir, script)):
+            script_file_path = os.path.join(scripts_dir, script)
+            if os.path.isfile(script_file_path) and not is_hidden_file(script):
```

Comment:
Please use the stanza in https://github.com/pybuilder/pybuilder/pull/476/commits/a3f79fd78e47ad19bed850648a54de80c6f29466#diff-72f9577cabb10f76a3bf6bd78bdcd06fR112 with `HIDDEN_FILE_NAME_PATTERN`